packages/rum-core/src/domain/foregroundContexts.spec.ts

```diff
@@ -0,0 +1,280 @@
+import { RelativeTime, relativeNow, ServerDuration, toServerDuration, Duration } from '@datadog/browser-core'
+import { setup, TestSetupBuilder } from '../../test/specHelper'
+import { createNewEvent } from '../../../core/test/specHelper'
+import { InForegroundPeriod } from '../rawRumEvent.types'
+import { startForegroundContexts, ForegroundContexts } from './foregroundContexts'
+
+const FIVE_SECONDS_MS = 5_000 as Duration
+const TEN_SECONDS_MS = 10_000 as Duration
```

Comment:
It could be more valuable to name these variables with their purpose rather than their value.